bzt/modules/java.py

```diff
@@ -58,38 +58,61 @@ def __init__(self):
         self.working_dir = os.getcwd()
         self.target_java = "1.8"
         self.props_file = None
-        self.base_class_path = []
+        self.class_path = []
+        self._tools = []
```

Comment:
I would argue for `self.tools` for consistency (all other attributes are not private).